cardano_node_tests/tests/test_cli.py

```diff
@@ -165,6 +165,22 @@ def test_pretty_utxo(
 
         assert utxo_out == expected_out
 
+    @allure.link(helpers.get_vcs_link())
+    @pytest.mark.testnets
+    def test_txid_with_process_substitution(self):
+        """Check that it is possible to use 'transaction txid' using process substitution."""
+        cmd = (
+            f"txFileJSON=$(cat {DATA_DIR / 'unwitnessed.tx'});"
+            'cardano-cli transaction txid --tx-file <(echo "${txFileJSON}")'
+        )
+
+        try:
+            helpers.run_in_bash(command=cmd)
+        except AssertionError as err:
+            if "cardano-cli: TODO" in str(err):
+                pytest.xfail("Not possible to use process substitution - see node issue #4235")
```

Comment:
This test will fail on the error message on 1.35.4-rc1 and older. We want it to xfail there as well.